endpoint/contracts/examples/treasury/LockTreasury.sol

```diff
@@ -35,17 +35,35 @@ contract LockTreasury {
         string memory name = IERC20Metadata(token).name();
         string memory symbol = IERC20Metadata(token).symbol();
         bytes memory data = abi.encodeWithSelector(
-            bytes4(keccak256("mint(address,string,string,address,uint256)")),
+            bytes4(
+                keccak256("mint(address,string,string,address,uint256,uint256)")
```

Comment:
question just for clarity: which contract's function `mint` this is calling?

Reply:
mintTreasury